text/086-reduce-barriers-to-contributing-to-stdlib.md

```diff
@@ -0,0 +1,250 @@
+
+- The canonical role and name "stdlib"
+- Monorepo and unified distribution (with coq, or at keeping both theory together with associated plugins)
+- Requirement for passing CI and having code reviewed
+- Support for only one coq version in stdlib (perhaps make a separate repo for backports, but keep it small)
+- This CEP assumes the circular-dependency-avoidance strategy from #83, but not the broader vision there
+
+# Standards for Code Review (dfaft additions to  [Contributing Guide](https://github.com/coq/coq/blob/master/CONTRIBUTING.md#additional-notes-for-pull-request-reviewers-and-assignees))
+
+Every change to Coq should be solid, definitely useful to somebody, and plausibly useful to others.
+Not missing out on easy improvements is an aspiration but not a hard requirement, and we should not pursue perfection.
+However, different types of changes come with widely varying levels of associated risk of unforeseen issues,
+so each change should be reviewed in the context of its own potential upside and downside.
+In particular, easy-to-reverse changes changes should not require extensive motivation and deliberation.
+On the other hand, potentially hard-to-reverse changes should be discussed openly,
+and all reasoning for or against should be reflected on the the PR (or a CEP) before finalizing the decision.
+
+### Easily Reversible Changes for Lightweight Review
+
+- Additions of lemmas about existing definitions (if definitions with the same basename exist in CI, fully qualify references to them)
+- Additions of new definitions in new files, or as `Local` with `Private_` in the fully qualified name
+- `Local` tactics
+- Removals of infrequently used functionality that could be inlined to potential use cases
+
+### Changes That May Lead to Direct Rigid Uses
+
+Consideration of "do we want to recommend the way we are doing this?" is expected for:
+
+- Additions of importable definitions
+- Additions of importable tactics (regardless of implementation language)
+- Notations, #[export], #[global] in modules that contain definitions and lemmas -- when in doubt, put them in their own module
+
+### Pervasive Changes
+
+Thorough ahead-of-time planning and justification is expected for changes that users might not be able to adapt to, or whose reversal may soon become infeasible. This includes:
+- Changes to the term AST or effective convertibility relation
+- Elaboration frameworks (Function, Program, type classes in terms) -- for the purposes of tactic programming, the output of these tools is as special as the term AST
+- Introducing new dependencies
+
+## External Reviewers
+
+It may be the case that the Coq team is not available or equiped to review one change or another, even though the concept looks valuable.
+In this case anyone can suggest an external reviewer, and with the blessing of a component maintainer or core developer, this external reviewer can take care of the technical vetting aspects of the PR review process.
```

Comment:
It would be nice if the lines were limited to about 80 characters or so.  Some of them are so long it's a little hard to go the next line of text when reading.